unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/BuilderMode/Tests/BIWEntityHandlerShould.cs

```diff
@@ -4,19 +4,26 @@
 using System.Collections.Generic;
 using UnityEngine;
 using System.Linq;
+using DCL.Camera;
+using DCL.Components;
 using DCL.Models;
+using Newtonsoft.Json;
+using NSubstitute;
+using UnityEngine.TestTools;
 
 public class BIWEntityHandlerShould : IntegrationTestSuite_Legacy
```

Comment:
```suggestion
public class BIWEntityHandlerShould : IntegrationTestSuite
```

Why do we need the `_Legacy` suite? I don't see camera or character controller interactions here.

Reply:
There is a bug in the NSubstitute library where it is unable to mock up the entity and give an exception. I need to setup the environment in order to make the unit test. I will add a note here to ensure that we are aware of it and change it in the future